include/dlaf/lapack/tile.h

```diff
@@ -388,8 +387,10 @@ void assertExtendInfo(F assertFunc, cusolverDnHandle_t handle, CusolverInfo<T>&&
   DLAF_CUSOLVER_CALL(cusolverDnGetStream(handle, &stream));
   assertFunc(stream, info.info());
   // Extend info scope to the end of the kernel execution
-  pika::cuda::experimental::detail::get_future_with_event(stream)  //
-      .then(pika::launch::sync, [info = std::move(info)](pika::future<void>&&) {});
+  auto extend_info = [info = std::move(info)]() {};
+  pika::cuda::experimental::detail::get_future_with_event(stream) |
+      pika::execution::experimental::then(std::move(extend_info)) |
+      pika::execution::experimental::start_detached();
```

Comment:
To do: use `pika::cuda::detail::add_event_callback`? If I'm using `detail` might as well use it. However, would be best not to use `detail` in the first place... This might need some refactoring.